lib/GADS.pm

```diff
@@ -1602,18 +1602,22 @@ any ['get', 'post'] => '/user/:id' => require_any_role [qw/useradmin superadmin/
         }
     }
 
-    my $titles = [map { +{ label_html => $_->name, value => $_->id } } @{$userso->titles}];
+    my $titles = [ map { +{ label_plain => $_->name, value => $_->id } } @{ $userso->titles } ];
 
-    my $orgs = [map { +{ label_html => $_->name, value => $_->id } } @{$userso->organisations}];
+    my $orgs = [ map { +{ label_plain => $_->name, value => $_->id } } @{ $userso->organisations } ];
+
+    my $depts = [ map { +{ label_plain => $_->name, value => $_->id } } @{ $userso->departments } ];
+
+    my $teams = [ map { +{ label_plain => $_->name, value => $_->id } } @{ $userso->teams } ];
```

Comment:
Another thing I have just realised is that all of the above also needs to be passed/updated into the `user_overview` template in the `user_overview` route. As such, rather than repeat all the above code, I think it would be better to move the code into the `GADS::Users` class.

Reply:
Changed the "build" code for each attribute - although part of me wonders if this will "break" something else(?) If preferred, I'll give each it's own attribute seperate from the current ones

Reply:
Thanks, this is definitely the way to go. However, you are correct, I think it may have broken the "register" and "my account" page. Can you take a look at those please and fix as required please? It looks like it would be worth updating them to use the same select drop-down too.

It looks like the `/login` route also has them in the route but that they are not used in the template, can you check and remove from the controller as required please?

Reply:
Completed as above